src/components/ThemeProvider/stories/full.md

```diff
@@ -51,12 +51,11 @@ function ExampleComponent(props) {
 
 #### GearboxTheme
 
-The object that defines colours and some other CSS properties for `@cognite/gearbox` components.
+The objects that define colours and some other CSS properties for `@cognite/gearbox` components.
```

Comment:
Why the plural form?

Reply:
my bad, was left from two types overview. fixed